include/envoy/ssl/context_config.h

```diff
@@ -16,6 +17,40 @@
 namespace Envoy {
 namespace Ssl {
 
+class TlsCertificateConfigProvidersFactory {
+public:
+  virtual ~TlsCertificateConfigProvidersFactory() = default;
+
+  /**
+   * Create the certificate config providers factory
```

Comment:
Period at end of sentence. And based on the type, I think it should be "config provider factories." (switched the plural around)